.docksal/README.md

```diff
@@ -3,13 +3,14 @@
 ##Steps to setup
 
 1. clone repo -- will default SOE-develop-mc branch.
-2. Checkout branch, e.g., `git checkout -b SOE-jiraticket-shortdescription`
+2. Checkout new branch, e.g., `git checkout -b SOE-jiraticket-shortdescription`
 3. Edit the following files, which are located ./docksal
-    3. Add branch name to line six in Gruntfile.js.
-    4. Edit soe.aliases.drushrc.php, making sure the slugs path to build/html is correct.
-    5. Edit settings.local.php, updating the `base_url` var. 
-6. Run `fin init`
-    
+    1. Add branch name to line six in Gruntfile.js.
+    2. Edit soe.aliases.drushrc.php, making sure the slugs path to build/html is correct.
+    3. Optional: edit settings.local.php, updating the `base_url` var. Note that the `base_url` is set based on the directory name of this repository, e.g., `grunt-drupal-tasks.docksal`.
+4. If you use a key other than `id_rsa` to connect to Acquia, run `fin ssh-key add <keyname>` (e.g., `fin ssh-key add id_rsa_acquia_4096`)
```

Comment:
This does not persist. I'm not sure yet what causes it to go away - stopping the Docksal application, destroying the containers, or restarting my machine.

Reply:
HI John,

You'll need to add these keys to the global environment file. 

> For example, assuming you have a private SSH key $HOME/.ssh/acquia_key, you would define a variable:
> SECRET_SSH_KEY_ACQUIA='acquia_key'

Source: https://blog.docksal.io/loading-ssh-keys-in-docksal-containers-c6a48944ccc8

Reply:
Global environment file, which is tracked in Git, or in `.docksal/docksal-local.env`?

`.docksal/docksal-local.env` appears to be working; I will add that to `.gitignore`

Reply:
Sorry. I didn't see your comment before making my last one. See my last one for location of the global.

Also, you'll have to run this command: `fin project restart` after adding the keys.